From 6ae32ed336666f3015c2e9bc26c2b39aecb31cd3 Mon Sep 17 00:00:00 2001
From: binzhang <binzhang@codeaurora.org>
Date: Tue, 26 Jun 2018 14:07:45 +0800
Subject: [PATCH 22/37] Remove the property of touch optimization check.

Vendor related property check is no longer
suitable in FW codes, we just remove the property
check from now on, and with touch optimzation enable
by default.

Change-Id: I13c946bcc24418f0dd1e1d37afdefe17281c77ff
CRs-Fixed: 2267588
---
 core/java/android/view/Choreographer.java | 2 +-
 core/java/android/widget/AbsListView.java | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/core/java/android/view/Choreographer.java b/core/java/android/view/Choreographer.java
index 48099ab2f6b..0ff08226636 100644
--- a/core/java/android/view/Choreographer.java
+++ b/core/java/android/view/Choreographer.java
@@ -80,7 +80,7 @@ public final class Choreographer {
 
     // Prints debug messages about jank which was detected (low volume).
     private static final boolean DEBUG_JANK = false;
-    private static final boolean OPTS_INPUT = SystemProperties.getBoolean("persist.vendor.qti.inputopts.enable",false);
+    private static final boolean OPTS_INPUT = true;
 
     // Prints debug messages about every frame and callback registered (high volume).
     private static final boolean DEBUG_FRAMES = false;
diff --git a/core/java/android/widget/AbsListView.java b/core/java/android/widget/AbsListView.java
index 81f5412c4b6..d52c68cad23 100644
--- a/core/java/android/widget/AbsListView.java
+++ b/core/java/android/widget/AbsListView.java
@@ -111,8 +111,8 @@ public abstract class AbsListView extends AdapterView<ListAdapter> implements Te
 
     @SuppressWarnings("UnusedDeclaration")
     private static final String TAG = "AbsListView";
-    private static final boolean OPTS_INPUT = SystemProperties.getBoolean("persist.vendor.qti.inputopts.enable",false);
-    private static final String MOVE_TOUCH_SLOP = SystemProperties.get("persist.vendor.qti.inputopts.movetouchslop","0.6");
+    private static final boolean OPTS_INPUT = true;
+    private static final double MOVE_TOUCH_SLOP = 0.6;
     private static final double TOUCH_SLOP_MIN = 0.6;
     private static final double TOUCH_SLOP_MAX = 1.0;
 
@@ -899,7 +899,7 @@ public abstract class AbsListView extends AdapterView<ListAdapter> implements Te
         mTouchSlop = configuration.getScaledTouchSlop();
         mVerticalScrollFactor = configuration.getScaledVerticalScrollFactor();
         if (OPTS_INPUT) {
-            double touchslopprop = Double.parseDouble(MOVE_TOUCH_SLOP);
+            double touchslopprop = MOVE_TOUCH_SLOP;
             if (touchslopprop > 0) {
                 if (touchslopprop < TOUCH_SLOP_MIN) {
                     mMoveAcceleration = (int)(mTouchSlop * TOUCH_SLOP_MIN);
-- 
2.17.1

